docs/examples/vector_stores/AstraDBIndexDemo.ipynb

```diff
@@ -0,0 +1,209 @@
+{
```

Comment:
<p>Can you add in a cell just above a snippet like the one below for an "Open In Colab" button?</p><p> </p><pre class="ql-syntax" spellcheck="false">&lt;a href="https://colab.research.google.com/github/run-llama/llama_index/blob/main/docs/examples/vector_stores/AsyncIndexCreationDemo.ipynb" target="_parent"&gt;&lt;img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/&gt;&lt;/a&gt;
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/8777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@nerdai Good call, just added this :)